### DISCUSSION TITLE
Compatibility with IR3 and a few other things

### DISCUSSION THREAD

**Discussion by the-clorax** [Xman1109/Atomic_Overhaul#36] [General]
A while ago i made a mod called better nuclear, which you saw was a mess with some good ideas that this mod may have taken inspiration from (sorry if i sound rude saying that, i really like that you did a lot of what i wanted to do). i would like to create some a file for compatibility with IR3, as a better nuclear power system without kovarex enrichment is my main reason for making better nuclear.

the main change i have in mind to add compatibility is changing the empty fuel cell to have nothing besides {lead-plate-special, 10} be its only ingredient, removing the graphite recipe because IR3 already has a graphite recipe, and changing the recipe for the heat oven to {{heat-pipe, 4}, {copper-foam, 8}, (chromium-frame-small, 1}, {chromium-plate-heavy, 10}}, as well as lead-scrap and steel-scrap in the reprocessing recipes

also for realism it may be nice to have some inert material to the fuel rods or fuel cells like in vanilla factorio and real life, as uranium-238 gets neutron-bred into plutonium-239 and thorium-232 fuel rods build up a lot of U-233, making it often be a breeder reactor fuel to create U-233 fuel cells at the cost of thorium and storage of radioactive byproducts.

the research data is a nice idea, but it might be better as a component to alternative recipes for some science packs

**Comment by Xman1109** [comment]
I remember your mod, it inspired me and my friend to make a mod about nuclear power ourselves. So you are right, your work inspired our mod.

Of course, I will work on compatibility for IR3, but it could take a while, because the following week I am not home (passed my finals and getting on the last trip together with my class).

If you want, you can fork this repo and work on some changes yourself, which I can integrate into the mod on the weekend.

It isn't the first time I hear about making an own beaker texture for the research data, and I want to do that (probably planned for the next update). But I need a color that fits the factorio style, and I haven't thought too much about it yet.
